mitosheet/mitosheet/scheduling/schedule_utils.py

```diff
@@ -0,0 +1,280 @@
+
+from mitosheet.types import StepsManagerType
+
+def get_path_of_import_param_for_automation(
+        automation_base_folder: str,
+        original_value: str,
+    ) -> str:
+    file_name = os.path.basename(original_value)
+    return os.path.join(automation_base_folder, 'data', file_name)
+
+def get_data_files_for_automation(
+        steps_manager: StepsManagerType,
+        automation_base_folder: str,
+    ) -> Dict[str, str]:
+
+    data_files: Dict[str, str] = {}
+
+    # First, we get the CSV and Excel files the user has importer
+    import_params = [param for param in steps_manager.param_metadata if param['type'] == 'import']
+    original_paths = [param['original_value'] for param in import_params if 'file' in param['subtype'] and param['original_value'] is not None]
+
+    for original_path in original_paths:
+        new_path = get_path_of_import_param_for_automation(
+            automation_base_folder,
+            original_path
+        )
+        data_files[new_path] = open(original_path).read()
+
+    return data_files
+
+def get_minute_and_hour_from_time(time: str) -> Tuple[int, int]:
+    hour = time.split(':')[0]
+    minute = time.split(':')[1]
+
+    # Handle the timezone here. Github actions run in UTC, so we need to convert the time to UTC
+    # from the current timezone of the user
+    current_timezone = datetime.datetime.now(datetime.timezone.utc).astimezone().tzinfo
+    hour_and_minute = datetime.datetime.strptime(f'{hour}:{minute}', '%H:%M').time()
+
+    # Then, make a datetime object with the current timezone, and convert it to UTC
+    hour_and_minute = datetime.datetime.combine(datetime.date.today(), hour_and_minute)
+
+    # Then, we convert it to the current timezone, and then to UTC
+    hour_and_minute = hour_and_minute.replace(tzinfo=current_timezone)
+    hour_and_minute = hour_and_minute.astimezone(datetime.timezone.utc)
+
+    hour_int = hour_and_minute.hour
+    minute_int = hour_and_minute.minute
+
+    return hour_int, minute_int
+
+def get_cron_string_from_schedule(schedule: Dict[str, Any]) -> str:
+    if schedule['type'] == 'Every Day':
+        hour, minute = get_minute_and_hour_from_time(schedule['time'])
+        return f'{minute} {hour} * * *'
+
+    elif schedule['type'] == 'Every Week':
+        hour, minute = get_minute_and_hour_from_time(schedule['time'])
+        day_of_week = schedule['dayOfWeek']
+        return f'{minute} {hour} * * {day_of_week}'
+
+    elif schedule['type'] == 'Every Month':
+        hour, minute = get_minute_and_hour_from_time(schedule['time'])
+        day_of_month = schedule['dayOfMonth']
+        return f'{minute} {hour} {day_of_month} * *'
+
+    else:
+        raise Exception(f"Invalid schedule type: {schedule['type']}")
+    
+def get_automation_code(
+        steps_manager: StepsManagerType,
+        automation_name: str,
+        automation_base_folder: str,
+    ) -> str:
+
+    # First, we get the function name
+    # TODO: improve this
```

Comment:
Update all these things... we should have:

1. `Automation Name`
2. `Automation Name as Python Variable`
3. `Automation Name as Folder` -- maybe this is just (2)?